src/MainnetController.sol

```diff
@@ -199,18 +201,6 @@ contract MainnetController is AccessControl {
         emit MaxSlippageSet(pool, maxSlippage);
     }
 
-    function setUniswapV3Router(address router) external {
-        _checkRole(DEFAULT_ADMIN_ROLE);
-        uniswapV3Router = ISwapRouter(router);
-        emit UniswapV3RouterSet(router);
-    }
-
-    function setUniswapV3PositionManager(address positionManager) external {
-        _checkRole(DEFAULT_ADMIN_ROLE);
-        uniswapV3PositionManager = INonfungiblePositionManager(positionManager);
-        emit UniswapV3PositionManagerSet(positionManager);
-    }
-
```

Comment:
I might have missed something somewhere, but why are we removing the admin functions to set these addresses? If we do remove them, then they effectively become immutable in this contract, and we need to remove the router and positionManager zero address checks in the lib.

Reply:
https://github.com/grove-labs/grove-alm-controller/pull/66#discussion_r2535900083